pyrobot/plugins/account_profile.py

```diff
@@ -0,0 +1,91 @@
+"""Profile Updation Commands
+.pbio <Bio>
+.pname <Name>
+.ppic"""
+import logging
+import os
+from datetime import datetime
+
+from pyrogram import Client, Filters
+from pyrogram.api import functions, types
+from pyrogram.api.functions.account import UpdateProfile
+from pyrogram.api.functions.photos import UpdateProfilePhoto
+
+from pyrobot import COMMAND_HAND_LER, TMP_DOWNLOAD_DIRECTORY
+
+logging.basicConfig(format='[%(levelname) 5s/%(asctime)s] %(name)s: %(message)s',
+                    level=logging.WARNING)
+logger = logging.getLogger(__name__)
+
+@Client.on_message(Filters.command(r"pbio", ".")  & Filters.me)
```

Comment:
i dont think you need the r prefix. 

Also change the . To the Command Handler variable Name.